.github/workflows/covector.yml

```
@@ -0,0 +1,156 @@
+name: Covector
+on:
+  push:
+    branches:
+      - dev
+    paths-ignore:
+      - 'specs/**'
+      - 'documentation/**'
+  pull_request:
+    branches:
+      - dev
+    paths-ignore:
+      - 'specs/**'
+      - 'documentation/**'
+
+jobs:
+  status:
+    name: Status
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout the Source Code
+        uses: actions/checkout@v2
+
+      - name: Run Covector Status
+        uses: jbolda/covector/packages/action@covector-v0
+        id: covector
+        with:
+          command: status
+
+  version-or-publish:
+    name: Version-or-publish
+    runs-on: ubuntu-latest
+    outputs:
+      change: ${{ steps.covector.outputs.change }}
+      commandRan: ${{ steps.covector.outputs.commandRan }}
+      successfulPublish: ${{ steps.covector.outputs.successfulPublish }}
+
+    steps:
+      - name: Checkout the Source Code
+        uses: actions/checkout@v2
+
+      - name: Set Up Node.js 14
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+
+      - name: Install Dasel
+        run: brew install dasel
+
+      - name: Configure the Git User to Use
+        run: |
+          git config --global user.name "${{ github.event.pusher.name }}"
+          git config --global user.email "${{ github.event.pusher.email }}"
+
+      # Publish when no change file is present.
+      - id: covector
+        name: Run Covector Version-or-publish
+        uses: jbolda/covector/packages/action@covector-v0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          command: version-or-publish
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Create Pull Request with Versions Bumped
+        if: ${{ steps.covector.outputs.commandRan == 'version' }}
+        uses: iotaledger/create-pull-request@v3.4.1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          branch: release/version-updates
+          title: Apply Version Updates From Current Changes
+          commit-message: apply version updates
+          labels: version updates
+          body: ${{ steps.covector.outputs.change }}
+
+  nodejs-binding-prebuild:
+    name: Build Prebuilt Binaries for Node.js
+    if: ${{ needs.version-or-publish.outputs.successfulPublish == 'true' }}
+    needs: version-or-publish
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

```suggestion
        os: [ubuntu-18.04, macos-latest, windows-latest]
```

I think we build on Ubuntu 18.04 to avoid issues with glibc (#576)